target/linux/ath79/image/generic.mk

```diff
@@ -891,6 +891,18 @@ define Device/engenius_ens202ext-v1
 endef
 TARGET_DEVICES += engenius_ens202ext-v1
 
+define Device/engenius_enstationac-v1
+  $(Device/engenius_loader_okli)
+  SOC := qca9557
+  DEVICE_MODEL := EnStationAC
+  DEVICE_VARIANT := v1
+  DEVICE_PACKAGES := ath10k-firmware-qca988x-ct rssileds
```

Comment:
You are missing kmod-ath10k-ct or -smallbuffers variant here, adding just the firmwaer.

Reply:
hmm...I see that it selects kmod-ath10k automatically as a dependency...

why is kmod-ath10k-ct not set as dependency for ath10k-firmware-qca988x-ct?

Reply:
> hmm...I see that it selects kmod-ath10k automatically as a dependency...

Where?

> why is kmod-ath10k-ct not set as dependency for ath10k-firmware-qca988x-ct?

E.g. because you might want kmod-ath10k-ct-smallbuffers instead.

Reply:
Sorry I'm confusing with my custom build

kmod-ath10k is selected automatically when I have ath10k-firmware-qca988x (not ct version) I think...